torch/_dynamo/variables/builtin.py

```diff
@@ -1365,6 +1365,11 @@ def _comparison(self, tx, left, right):
 
         def _unimplemented():
             unimplemented(f"comparison {typestr(left)} {op} {typestr(right)}")
+        
+        if op.__name__ == "is_":
+            # If the two objects are of different type, we can safely return False
+            if type(left) is not type(right):
+                return ConstantVariable.create(False)
```

Comment:
This is incorrect. `is_` doesn't just mean type. It actually means equality of objects.

```
>>> MyClass(1) is MyClass(1)
False
>>> type(MyClass(1)) == type(MyClass(1))
True
```

Reply:
In particular, this is not good enough. SymNodeVar is never of type ListVariable. Lol.